MultiVersions/Forge/main/letsdomeadow/lang/zh_tw.json

```
@@ -0,0 +1,280 @@
+  "item.meadow.cheesecake_slice": "乳酪蛋糕切片",
+  "item.meadow.cooked_bear_meat":"烤熊肉",
+  "item.meadow.cookie_cow_spawn_egg":"餅乾牛 生怪蛋",
+  "item.meadow.cream_cow_spawn_egg":"奶油牛 生怪蛋",
+  "item.meadow.dairy_cow_spawn_egg":"奶牛 生怪蛋",
+  "item.meadow.dark_cow_spawn_egg":"深色牛 生怪蛋",
+  "item.meadow.flecked_sheep_spawn_egg":"斑點羊 生怪蛋",
+  "item.meadow.fur_boots": "毛皮靴子",
+  "item.meadow.fur_chestplate": "毛皮胸甲",
+  "item.meadow.fur_helmet": "毛皮頭盔",
+  "item.meadow.fur_leggings": "毛皮護腿",
+  "item.meadow.fuzzy_sheep_spawn_egg":"絨毛羊 生怪蛋",
+  "item.meadow.highland_cattle_spawn_egg":"高地牛 生怪蛋",
+  "item.meadow.horned_sheep_spawn_egg":"角羊 生怪蛋",
+  "item.meadow.ingredient.tooltip": "合成原材料",
+  "item.meadow.inky_sheep_spawn_egg":"墨水羊 生怪蛋",
+  "item.meadow.jug.tooltip": "解除負面狀態效果",
+  "item.meadow.long_nosed_sheep_spawn_egg":"長鼻羊 生怪蛋",
+  "item.meadow.meadow_chicken_spawn_egg":"高山雞 生怪蛋",
+  "item.meadow.patched_sheep_spawn_egg":"補丁羊 生怪蛋",
+  "item.meadow.piece_of_amethyst_cheese": "紫水晶乳酪切片",
+  "item.meadow.piece_of_buffalo_cheese": "水牛乳酪切片",
+  "item.meadow.piece_of_cheese": "乳酪切片",
+  "item.meadow.piece_of_goat_cheese": "山羊乳酪切片",
+  "item.meadow.piece_of_grain_cheese": "穀物乳酪切片",
+  "item.meadow.piece_of_sheep_cheese": "綿羊乳酪切片",
+  "item.meadow.piece_of_warped_cheese": "扭曲乳酪切片",
+  "item.meadow.pinto_cow_spawn_egg":"花斑牛 生怪蛋",
+  "item.meadow.raw_bear_meat": "生熊肉",
+  "item.meadow.rennet": "凝乳酶",
+  "item.meadow.rocky_sheep_spawn_egg":"岩石羊 生怪蛋",
+  "item.meadow.sunset_cow_spawn_egg":"黃昏牛 生怪蛋",
+  "item.meadow.tooltip.damage": "攻擊傷害",
+  "item.meadow.umbra_cow_spawn_egg":"陰影牛 生怪蛋",
+  "item.meadow.wooly_cow_spawn_egg": "長毛牛 生怪蛋",
+  "item.meadow.water_buffalo_spawn_egg":"水牛 生怪蛋",
+  "item.meadow.watering_can": "澆水壺",
+  "item.meadow.wooden_amethyst_milk_bucket": "木桶裝紫水晶...鮮奶？",
+  "item.meadow.wooden_bucket": "木桶",
+  "item.meadow.wooden_buffalo_milk_bucket": "水牛鮮奶木桶",
+  "item.meadow.wooden_goat_milk_bucket": "山羊鮮奶木桶",
+  "item.meadow.wooden_grain_milk_bucket": "穀物鮮奶木桶",
+  "item.meadow.wooden_milk_bucket": "鮮奶木桶",
+  "item.meadow.pine_chest_boat": "儲物箱松木船",
+  "item.meadow.pine_boat": "松木船",
+  "block.meadow.pine_hanging_sign": "懸掛式松木告示牌",
+  "block.meadow.pine_sign": "松木告示牌",
+  "item.meadow.wooden_sheep_milk_bucket": "綿羊鮮奶木桶",
+  "item.meadow.wooden_warped_milk_bucket": "扭曲鮮奶木桶",
+  "item.meadow.ingredient.tooltipitem.meadow.wooden_warped_milk_bucket": "合成原材料",
+  "item.meadow.ingredient.tooltipitem.meadow.wooden_sheep_milk_bucket": "合成原材料",
+  "item.meadow.ingredient.tooltipitem.meadow.wooden_milk_bucket": "合成原材料",
+  "item.meadow.ingredient.tooltipitem.meadow.wooden_grain_milk_bucket": "合成原材料",
+  "item.meadow.ingredient.tooltipitem.meadow.wooden_goat_milk_bucket": "合成原材料",
+  "item.meadow.ingredient.tooltipitem.meadow.wooden_buffalo_milk_bucket": "合成原材料",
+  "item.meadow.ingredient.tooltipitem.meadow.wooden_amethyst_milk_bucket": "我們不知道這是誰想出的主意...",
+  "item.meadow.wooden_water_bucket": "水木桶",
+  "creativetab.meadow.tab": "Meadow",
```

我在想要不要將 `Let's Do` 譯作 `共度時光`，而這款模組是該系列中的「牧場」主題。
```suggestion
  "creativetab.meadow.tab": "共度時光｜牧場",
```